content/src/fieldConfig/config.json

```diff
@@ -26,28 +26,36 @@
     "START": "Start",
     "OPERATE": "Operate"
   },
-  "betaBar": {
+  "feedbackModal": {
     "betaModalIntercomButtonText": "Speak with a NJ-Based Business Expert",
-    "betaModalFeedbackButtonText": "Share Feedback",
     "featureRequestModalBodyText": "**How can we improve the Navigator?**",
     "successfulSubmissionModalButtonText": "Submit Another Request",
-    "featureRequestModalInlineErrorText": "This field is required.",
+    "feedbackModalInlineErrorText": "This field is required.",
     "successfulSubmissionModalHeadingText": "Thank you!",
     "betaModalBugReportButtonText": "Report an Issue",
     "betaFormLinkOwning": "https://forms.business.nj.gov/feedback/owning",
     "unsuccessfulSubmissionAlertText": "There was an error. Please try again or email your issue to xyz@xyz.gov.",
-    "featureRequestModalSubmitButtonText": "Submit",
-    "featureRequestModalPlaceholderText": "Start typing here...",
+    "feedbackModalSubmitButtonText": "Submit",
+    "feedbackModalPlaceholderText": "Start typing here...",
     "betaModalBodyText": "Get in contact, report a bug, or share your feedback.",
-    "featureRequestModalCancelButtonText": "Cancel",
+    "feedbackModalCancelButtonText": "Cancel",
     "betaFormLinkStarting": "https://forms.business.nj.gov/feedback/starting",
     "betaModalFeatureRequestButtonText": "Request a Feature",
-    "betaMainText": "This site is in beta.",
     "successfulSubmissionModalBodyText": "Your request has been submitted.",
     "featureRequestModalSecondBodyText": "Describe below the information or feature you are looking for.",
     "betaModalTitle": "Let us know what you think",
     "featureRequestModalHeadingText": "Request a Feature",
-    "betaModalButtonText": "Share your feedback!"
+    "reportBugModalHeadingText": "Report an Issue",
+    "betaModalFeedbackButtonText": "Share Feedback",
+    "reportBugModalIssueSummaryBodyText": "**What were you trying to do?**",
+    "reportBugModalIssueSummaryAdditionalBodyText": "What is the issue you experienced? (i.e. I was not able to log in.)",
+    "reportBugModalIssueDetailBodyText": "**What was the issue you encountered?**",
+    "reportBugModalIssueDetailAdditionalBodyText": "Walk us through, step-by-step what happened. (i.e. This is my first time trying to create an account. When I was on the landing page I clicked “Get Started” but then nothing happened.)"
+  },
+  "bugBar": { "bugBarText": "Report an issue." },
```

Comment:
nitpick, but this is a button, not a Bar, right?

Reply:
reading more, I think I understand that sometimes the "beta bar" is actually just "report an issue" an that's why it's Bar here, right?